app/controllers/catalog_controller.rb

```diff
@@ -150,6 +150,17 @@ def catalog_item_types
     }
   end
 
+  def generic_item_subtypes
+    {
+      "custom"          => "Custom",
+      "vm"              => "VM",
+      "playbook"        => "Playbook",
+      "hosted_database" => "Hosted Database",
+      "load_balancer"   => "Load Balancer",
+      "storage"         => "Storage"
+    }
```

Comment:
@gmcculloug Yes, if the hash values are to be propagated into the UI at some point, then
they need to be inside a `_()`. Although I don't see where the `generic_item_subtypes()`
routine is being used.